deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,381 @@
+; Shows a lot of debug information while compiling
+; This can be removed once stable.
+!verbose 4
+;--------------------------------
+!define SF_USELECTED  0 ; used to check selected options status, rest are inherited from Sections.nsh
+
+    !include "MUI2.nsh"
+	!include "Sections.nsh"
+	!include "LogicLib.nsh"
+
+; Global variables that we'll use
+    Var _JELLYFINVERSION_
+    Var _JELLYFINDATADIR_
+	Var _INSTALLSERVICE_
+	Var _SERVICESTART_
+	Var _NETWORKSERVICEACCOUNT_
+	Var _EXISTINGINSTALLATION_
+	Var _EXISTINGSERVICE_
+	Var _CUSTOMDATAFOLDER_
+
+!if ${NSIS_PTR_SIZE} > 4
+  !define BITS 64
+  !define NAMESUFFIX " (64 bit)"
+!else
+  !define BITS 32
+  !define NAMESUFFIX ""
```

Comment:
You can put ` (32-bits)` here. Frankly 64-bit should be default since 64-bit ffmpeg is so much faster than the 32-bit version. (Extra registers really help encoding speeds)

Reply:
yeah, this bit needs to be tied up with packaging the right binaries. I have to study the build system a bit to figure that out.

Reply:
@crankdoofus Any luck on this bit? I do agree, we should push for 64-bit (even though I have no skin in the race for Windows haha) as the "default" and 32-bit as the anomaly worthy of a title. But that's just a minor quibble.

Reply:
There are multiple ways nsis can be configured:
1. NSIS can package both the binaries and install the ones relevant to the destination arch type. 
2. NSIS can create two packages depending on compiled binaries.

I assumed that the build is run using build-jellyfin.ps1 and since NSIS needs to be executed on the destination system (native windows) to compile a binary installer, it was easy to integrate.
I realised I was wrong and that the build is cross compiled on a Linux system (using dotnet publish) which makes it trickier to run NSIS. So the solution, per my limited knowledge, is to either 
1. Split the build system to compile (and package) on a native windows os OR
2. Find a way to execute NSIS on non windows environment.

Both these options stretch my current technical skills and it's taking me a while to close this.

I believe if someone can help out with solution (1) above, my changes won't take long.

Your thoughts @joshuaboniface, @EraYaN ?

Reply:
I think 1 is valid for now, we can automate that separately (or just do it manually on releases) for now. 2 is the ideal way forwards, but we can evaluate that separately later!